source/indexes/create-search-index.txt

```diff
@@ -0,0 +1,67 @@
+.. _compass-create-search-index:
+
+============================
+Create an Atlas Search Index
+============================
+
```

Comment:
Is this page intentionally missing an "On this page" side nav?

```suggestion
.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol
```

Reply:
Oops, thanks. Fixed. To my knowledge `default-domain` no longer does anything, so leaving that out.